MacOSX/README.md

```diff
@@ -24,16 +24,24 @@ The homebrew package manager is the packagemanger of choice.
 
 Important script vars:
 
-QMSDEVDIR               (mandatory: the build environment)
-BUILD_RELEASE_DIR       (optional: location where the bundles are put)
-XCODE_PROJECT           (optional: if set to 1, create an XCode project instead of building QMS)
-BUILD_GDAL              (optional: if set to 1, build GDAL instead of using the homebrew package)
+- QMSDEVDIR               (mandatory: the build environment)
+- BUILD_RELEASE_DIR       (optional: location where the bundles are put)
+
+Parameters to configure build:
+- XCODE_PROJECT           (optional: if set, create an XCode project instead of building QMS)
+                          can be set with -x on commandline
+- BUILD_GDAL              (optional: if set, build GDAL instead of using the homebrew package)
+                          can be set with -g on commandline
+- BREW_PACKAGE_BUILD      if set creates QMS as an app relying on brew packages on runtime
+                          homebrew packages are listed in install-brew-packages.sh
+                          can be set with -b on commandline
 
 
 To run the complete build process:
 1. Create a dir cd into this directory. This dir will be referenced as $QMSDEVDIR
-2. Download https://github.com/d029940/qmapshack/blob/dev/MacOSX/QMS_build_files/1st_QMS_start.sh and put it into this dir
-4. run "sh 1st_QMS_start.sh | tee log.txt"
+2. clone git repo https://github.com/d029940/qmapshack.git
```

Comment:
Is it Ok pointing to the fork, once merged to dev?

`https://github.com/d029940/qmapshack.git`
or
`https://github.com/Maproom/qmapshack.git`

Reply:
Good point. Just updated it.